package.json

```diff
@@ -4736,48 +4736,6 @@
       {
         "language": "csharp",
         "scopes": {
-          "namespace": [
-            "entity.name.type.namespace.cs"
-          ],
-          "type": [
-            "entity.name.type"
-          ],
-          "class": [
-            "entity.name.type.class.cs"
-          ],
-          "enum": [
-            "entity.name.type.enum.cs"
-          ],
-          "interface": [
-            "entity.name.type.interface.cs"
-          ],
-          "struct": [
-            "entity.name.type.struct.cs"
-          ],
-          "typeParameter": [
-            "entity.name.type.type-parameter.cs"
-          ],
-          "parameter": [
-            "entity.name.variable.parameter.cs"
-          ],
-          "variable": [
-            "entity.name.variable.local.cs"
-          ],
-          "property": [
-            "variable.other.property.cs"
-          ],
-          "enumMember": [
-            "variable.other.enummember.cs"
-          ],
-          "method": [
-            "entity.name.function.member.cs"
-          ],
-          "macro": [
-            "keyword.preprocessor.cs"
-          ],
-          "keyword": [
```

Comment:
same with keyword here.

Reply:
If `keyword.cs` doesn't work on the other themes - try just 'keyword'.  It looks like it should have the correct color in the theme json
https://github.com/dotnet/vscode-csharp/blob/main/themes/vs2019_dark.json#L311

Reply:
Okay, thank you, I will update and will double check them one by one and compare the outputs.

Reply:
Added this back in, looks ok now, please see my latest comment with comparisons.